### PR TITLE
Harinder kaur

### PR DESCRIPTION
**Pull request by HarinderKaur91**

### CODE REVIEW

File: src/main/java/com/naveenautomation/Base/TestBase.java [hunk 1]
```diff
@@ -15,16 +15,11 @@
...
- * This is a base class
- * 
- */
-
-	public static WebDriver driver;
-	public Browsers DEFAULT_BROWSER = Browsers.GOOGLE_CHROME;
-	public static WebdriverEvents events = new WebdriverEvents();
-	public EventFiringWebDriver eventFiringWebDriver;
+	
+	  public static WebDriver driver; public Browsers DEFAULT_BROWSER =
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Base/TestBase.java:19]: this is not correct, why u want to change it in this order

File: src/main/java/com/naveenautomation/Pages/MonitorsPage.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import com.naveenautomation.Browsers.ProxyDriver;
+
+public class MonitorsPage extends Page{
+
+	
+	private static final String PAGE_URL="/category&path=25_28";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/MonitorsPage.java:11]: we can't have page with &path=25_28"

File: src/test/java/com/naveenautomation/tests/MyWishListTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import com.naveenautomation.Base.TestBase;
+import com.naveenautomation.Pages.AccountLoginPage;
+import com.naveenautomation.Pages.MonitorsPage;
+import com.naveenautomation.Pages.MyAccountPage;
+import com.naveenautomation.Pages.MyWishListPage;
+import com.naveenautomation.Pages.MyWishListPage.WishList;
+
+public class MyWishListTest extends TestBase {
+
+	SoftAssert softAssert;
```
Comment (PivotITAcademy) [inline src/test/java/com/naveenautomation/tests/MyWishListTest.java:17]: Add access modifier

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]
Plz do the necessary changes

File: src/main/java/com/naveenautomation/Base/TestBase.java (hunk 1 above)
Comment (HarinderKaur91) [reply src/main/java/com/naveenautomation/Base/TestBase.java]: I have changed the order now.

File: src/main/java/com/naveenautomation/Pages/MonitorsPage.java (hunk 2 above)
Comment (HarinderKaur91) [reply src/main/java/com/naveenautomation/Pages/MonitorsPage.java:11]: This is page url and it is unique and required to load the page.

File: src/test/java/com/naveenautomation/tests/MyWishListTest.java (hunk 3 above)
Comment (HarinderKaur91) [reply src/test/java/com/naveenautomation/tests/MyWishListTest.java]: Access modifier added.

Comment (HarinderKaur91) [reply src/test/java/com/naveenautomation/tests/MyWishListTest.java]: Access modifier added.

File: src/main/java/com/naveenautomation/Pages/MonitorsPage.java (hunk 2 above)
Comment (PivotITAcademy) [reply src/main/java/com/naveenautomation/Pages/MonitorsPage.java:11]: &path=25_28 is specific to a product and not to page , remove this

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]

File: src/main/java/com/naveenautomation/Pages/MonitorsPage.java (hunk 2 above)
Comment (HarinderKaur91) [reply src/main/java/com/naveenautomation/Pages/MonitorsPage.java:11]: I have checked it again. &path=25_28 is not referring to any product .It is specific to monitors page.

Comment (HarinderKaur91) [reply src/main/java/com/naveenautomation/Pages/MonitorsPage.java]: I have now defined a method which will verify the static part of page url.

File: src/main/java/com/naveenautomation/Base/TestBase.java [hunk 4]
```diff
@@ -4,48 +4,37 @@
...
+	/*
+	 * This is a base class
+	 */
 
-/*
- * This is a base class 
- */
 	public static WebDriver driver;
 	public Browsers DEFAULT_BROWSER = Browsers.GOOGLE_CHROME;
-	public static WebdriverEvents events = new WebdriverEvents();
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Base/TestBase.java:23]: Why are u removing this

File: src/main/java/com/naveenautomation/Pages/MyWishListPage.java [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+				return headers.indexOf(webElement);
+			}
+		}
+		System.out.println("Column does not exist.....");
+		return -1;
+	}
+
+	public enum WishList {
+
+		IMAGE("Image"), PRODUCTNAME("Product Name"), MODEL("Model"), STOCK("Stock"), UNITPRICE("Unit Price"),
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/MyWishListPage.java:55]: enums should come one under the other, that is how they should be kept

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]

File: src/main/java/com/naveenautomation/Base/TestBase.java (hunk 4 above)
Comment (HarinderKaur91) [reply src/main/java/com/naveenautomation/Base/TestBase.java:23]: added

File: src/main/java/com/naveenautomation/Pages/MonitorsPage.java (hunk 2 above)
Comment (HarinderKaur91) [reply src/main/java/com/naveenautomation/Pages/MonitorsPage.java]: dynamic part of url removed

**Review by PivotITAcademy — APPROVED** [review]
LGTM

**Pull request merged**